scripts/run-nexus.sh

```diff
@@ -49,8 +49,9 @@ function runJsonScript {
     curl ${INSECURE} -X POST -sSf \
         --user "${ADMIN_USER}:${ADMIN_PASSWORD}" \
         --header 'Content-Type: application/json' \
-        "${NEXUS_URL}/service/rest/v1/script" -d @${SCRIPT_DIR}/nexus/"${jsonScriptName}".json
+        "${NEXUS_URL}/service/rest/v1/script" -d @"${SCRIPT_DIR}"/nexus/"${jsonScriptName}".json
     echo "running ${jsonScriptName}"
+    # shellcheck disable=SC2086
```

Comment:
I needed to disable this line for spellcheck because it was complaining about missing double quotes around `${runParams}` but using double quotes resulted in a curl error that I was not able to fix.

Reply:
I have seen this "problem" in other scripts as well and think it is kind of a weak spot of spellcheck as sometimes you actually want no quotes ...

Reply:
I am noticing this myself now as I am now backtracking every bug that is currently occuring due to the double quotes and disabling the rules or fixing the issues.
I still think shellcheck is valuable, you just need to be very careful not to break anything with its recommendations and not just blindly apply all of them as I did in the beginning thinking it couldn't do any harm